sys/net/gnrc/network_layer/ndp/internal/gnrc_ndp_internal.c

```diff
@@ -133,7 +131,7 @@ void gnrc_ndp_internal_set_state(gnrc_ipv6_nc_t *nc_entry, uint8_t state)
             mutex_unlock(&ipv6_iface->mutex);
             break;
 
-#ifdef ENABLE_DEBUG
+#if ENABLE_DEBUG
```

Comment:
I think you can remove this check since the case would be empty. I guess the compiler will optimize this.

Reply:
done